src/GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Tests/Functions/ProcessNextDataSetVersionMappingsFunctionTests.cs

```diff
@@ -609,7 +627,13 @@ await AddTestData<PublicDataDbContext>(context =>
 
             // Some source location options have no equivalent candidate to be mapped to, thus
             // resulting in a major version update.
-            Assert.Equal("2.0", updatedMappings.TargetDataSetVersion.Version);
+            Assert.Equal("2.0.0", updatedMappings.TargetDataSetVersion.SemVersion());
+
+            var updatedReleaseFile = await GetDbContext<ContentDbContext>()
+                .ReleaseFiles
+                .SingleAsync(rf => rf.PublicApiDataSetId == updatedMappings.TargetDataSetVersion.DataSetId);
+
+            Assert.Equal(updatedReleaseFile.PublicApiDataSetVersion, updatedMappings.TargetDataSetVersion.SemVersion());
```

Comment:
Looks like there's a mismatch between `Assert.Equal` `expected` and `actual` parameters here? Same comment applies elsehwere in this class.